pkg/rpctype/rpc.go

```diff
@@ -80,6 +80,8 @@ func NewRPCClient(addr string, timeScale time.Duration) (*RPCClient, error) {
 	if err != nil {
 		return nil, err
 	}
+	// Note: SetDeadline is not implemented on fuchsia, so don't fail on error.
+	conn.SetDeadline(time.Now().Add(3 * time.Minute * timeScale))
```

Comment:
Won't all fuzzer fail after 3 minutes?
This is absolute deadline, not relative per-call.

Reply:
Indeed, thanks for pointing it out.